### PR TITLE
Milestone 3 : Clear To-do

### PR DESCRIPTION
**Pull request by Cabraham1**
### In this milestone, I achieved the following...

- Add event to the clear button

- Select and clear all todo

### CODE REVIEW

File: src/clear.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/* eslint-disable */
```
Comment (amon-cofie) [inline src/clear.js]: You have done great work on this project and all your functionalities work as they are supposed to. However, while disabling linters might seem convenient to get the work finished, this action may cause you to overlook certain errors that could cause major problems when you have to consider scalability. I suggest that you remove this line and manually fix these linter issues 👍

**Review by amon-cofie — CHANGES_REQUESTED** [review]
Hi @Cabraham1 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
🚀 No linter errors
🚀 Good documentation
🚀 Good commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

[OPTIONAL] You have mentioned things you did in your PR summary 👍 but I suggest you try to make it as detailed as you possibly can. This does not only give the code reviewer insight about the things you did in your pull request but also helps you when you have to refer to this project in the future. 😉 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/clear.js [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
+    if (e.classList.contains('list-selected-item')) {
       deleteTask(e);
     }
   });
   let count = 0;
   const data = Array.from(localData).filter((e) => e.completed === false);
+  // eslint-disable-next-line no-return-assign, no-multi-assign
   data.map((e) => (e.index = count += 1));
   setStorage(data);
 };
```
Comment (TadesseDev) [inline src/clear.js]: - [ ] Dear @Cabraham1, I notice you have removed some of ignored linter tules as requested by the previous reviewer. But while doing that I also notice you have disabled `linters` somewhere else too. Remember `linters` are here to help us write clean code and we should use them accordingly. Please remove all linter-disabled lines.

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @Cabraham1 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

199733072-bdacca19-8df9-43a7-be93-3aff2f1c15c8

 ## Highlights
 - [ ] Good job making most of the changes requested by the previous reviewer. 🙏

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by RaoAkif — APPROVED** [review]
## STATUS: APPROVED 🎉🎉🎉

Hi @Cabraham1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approve-image

## Highlights
✔ Linters are on point 
✔ Splendid work with the README file
✔ Great use of GitHub flow
✔ Good code design and implementation

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Cabraham1** [comment]
thank @RaoAkif  you for reviewing and approving my code

**Pull request merged**